Client/Frontend/Home/HomePanelViewController.swift

```diff
@@ -26,6 +28,8 @@ protocol HomePanel: class {
 
 @objc
 protocol HomePanelDelegate: class {
+    func homePanelDidRequestToSignIn(homePanel: HomePanel)
```

Comment:
Do we think that ever implementer of this delegate will need these two methods? What do you think about making them optional?